moveit_ros/hybrid_planning/test/launch/hybrid_planning.launch.py

```diff
@@ -0,0 +1,199 @@
+        namespace="/",
+        package="rclcpp_components",
+        executable="component_container",
+        composable_node_descriptions=[
+            ComposableNode(
+                package="moveit_hybrid_planning",
+                plugin="moveit_hybrid_planning::GlobalPlannerComponent",
+                name="global_planner",
+                parameters=[
+                    global_planner_param,
+                    robot_description,
+                    robot_description_semantic,
+                    kinematics_yaml,
+                    ompl_planning_pipeline_config,
+                ],
+            ),
+            ComposableNode(
+                package="moveit_hybrid_planning",
+                plugin="moveit_hybrid_planning::LocalPlannerComponent",
+                name="local_planner",
+                parameters=[
+                    local_planner_param,
+                    robot_description,
+                    robot_description_semantic,
+                    kinematics_yaml,
+                ],
+            ),
+            ComposableNode(
+                package="moveit_hybrid_planning",
+                plugin="moveit_hybrid_planning::HybridPlanningManager",
+                name="hybrid_planning_manager",
+                parameters=[hybrid_planning_manager_param],
+            ),
+        ],
+        output="screen",
+    )
+
+    # RViz
+    rviz_config_file = (
+        get_package_share_directory("moveit_hybrid_planning")
+        + "/config/hybrid_planning_demo.rviz"
+    )
+    rviz_node = Node(
+        package="rviz2",
+        executable="rviz2",
+        name="rviz2",
+        output="log",
+        arguments=["-d", rviz_config_file],
+        parameters=[robot_description, robot_description_semantic],
+    )
+
+    # Static TF
+    static_tf = Node(
+        package="tf2_ros",
+        executable="static_transform_publisher",
+        name="static_transform_publisher",
+        output="log",
+        arguments=["0.0", "0.0", "0.0", "0.0", "0.0", "0.0", "world", "panda_link0"],
+    )
+
+    # Publish TF
+    robot_state_publisher = Node(
+        package="robot_state_publisher",
+        executable="robot_state_publisher",
+        name="robot_state_publisher",
+        output="both",
+        parameters=[robot_description],
+    )
+
+    # ros2_control using FakeSystem as hardware
+    ros2_controllers_path = os.path.join(
+        get_package_share_directory("moveit_hybrid_planning"),
+        "config",
+        "demo_controller.yaml",
+    )
+    ros2_control_node = Node(
+        package="controller_manager",
+        executable="ros2_control_node",
+        parameters=[robot_description, ros2_controllers_path],
+        output={
+            "stdout": "screen",
+            "stderr": "screen",
+        },
+    )
+
+    # Load controllers
+    load_controllers = []
+    for controller in [
+        "joint_state_controller",
+        "panda_joint_group_position_controller",
```

Comment:
I would move to using the `panda_arm_controller` here as the default, because that is what the rest of the MoveIt tutorials and demos use

Reply:
I am not sure if the joint_trajectory_controller is well suited for high-frequent goal updates as it is the case for the replanning demo with point-wise trajectory forwarding. What are your experiences with the controller in the demo setup?